http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/UnusedChannelExceptionHandler.java

```diff
@@ -60,8 +62,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
                                 "Java SDK, where a future was not completed while the channel was in use. The channel has " +
                                 "been closed, and the future will be completed to prevent any ongoing issues.", cause);
                 executeFuture.get().completeExceptionally(cause);
-
-            } else if (cause instanceof IOException) {
+            } else if (isNettyIoException(cause)) {
```

Comment:
I believe NettyIOException is intended to check here, can you confirm? @millems

Reply:
Adding `ReadTimeoutException` and `WriteTimeoutException` to the check makes sense. I thought we had removed those handlers at this point, though.